packages/common/src/Entity.ts

```diff
@@ -201,7 +201,7 @@ Entity.prototype.publish("padding", 8, "number", "padding");
 
 Entity.prototype.publish("annotationPaddingPercent", 38, "number", "annotationPaddingPercent");
 Entity.prototype.publish("annotationDiameter", 14, "number", "Annotation Diameter");
-Entity.prototype.publish("annotationIcons", [], "array", "annotationIcons");
+Entity.prototype.publish("annotationIcons", [], "array", "annotationIcons"); // TODO: is this really used?
```

Comment:
Should be - its the array of small icons - I would have thought it would have been a "widgetArray" though?